scripts/deployL2-it.js

```diff
@@ -12,7 +12,7 @@ const gasPrice = null;
 
 const config = [
   17, // maxKvSizeBits, 131072
-  32, // shardSizeBits ~ 4G
+  30, // shardSizeBits ~ 1G
```

Comment:
Please note that you also need to update the prepaidAmount if the shard size is changed.